4.analyze-components/scripts/util.py

```diff
@@ -110,7 +128,11 @@ def get_svcca_across_algorithm_stability(z_dict, algorithms,
     return pd.DataFrame(output_list, columns=out_cols)
 
 
-def get_svcca_across_z_stability(z_dict_a, z_dict_b, train_or_test='train'):
+def get_svcca_across_z_stability(z_dict_a,
+                                 z_dict_b,
+                                 algorithm,
+                                 train_or_test=None,
+                                 threshold=0.98):
     """
```

Comment:
You might be interested in this:  https://github.com/ambv/black

Daniel mentioned this new library to help formatting your code that you can incorporate using a hook.

Reply:
nice! I like it